apis/config/v1beta1/configuration_types.go

```diff
@@ -199,6 +202,20 @@ type WaitForPodsReady struct {
 	RequeuingTimestamp *RequeuingTimestamp `json:"requeuingTimestamp,omitempty"`
 }
 
+type MultiKueue struct {
+	// GCTimeout defines the timeout between two consecutive garbage collection runs.
+	// Defaults to 1min. If 0, the garbage collection is disabled.
+	// +optional
+	GCTimeout *metav1.Duration `json:"gcTimeout,omitempty"`
```

Comment:
Please rename to `GCInterval` or `GCPeriod`.

Reply:
why?

Reply:
Because the words `Interval` and `Period` capture the periodic nature of the mechanism. 

Timeout suggests it is the maximal amount of time for the GC to complete a single cycle.

Note that in the code k8s the analogous parameter is called `gcCheckPeriod`.

Reply:
done